src/extensions/agent_modules/cloud_communication/iothub_client/diagnostics_component/diagnostics_devicename/inc/diagnostics_devicename.h

```diff
@@ -1,39 +0,0 @@
-/**
- * @file diagnostics_devicename.h
- * @brief Header file describing the methods for setting the static global device name for use in the diagnostic component
- *
- * @copyright Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- */
-#ifndef DIAGNOSTICS_DEVICENAMES_H
```

Comment:
There's a corresponding config in the main.c that uses this function. I didn't see it removed in the rest of the code submitted unless I missed something. 

Did we refactor it somewhere adn I just don't know where that is?